.github/workflows/tests_linux.yml

```diff
@@ -47,13 +47,14 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install dependencies
-        run: echo ${{ github.event_name }} && sudo apt-get update && sudo apt-get -y -q install cmake gcc-$GCC_VERSION  g++-$GCC_VERSION  ninja-build gcovr lcov
+        run: echo ${{ github.event_name }} && sudo apt-get update && sudo apt-get -y -q install cmake gcc-$GCC_VERSION  g++-$GCC_VERSION liblapack-dev ninja-build gcovr lcov
 
       - name: Build and run unit tests
         run: |
             cmake . -BBuild -G Ninja \
               -DCMAKE_BUILD_TYPE=Debug \
               -DBUILD_TESTS=ON \
+              -DENABLE_LAPACK=ON \
```

Comment:
do we need to test Lightning with `ENABLE_LAPACK=OFF` to ensure the support in both cases?

Reply:
Thanks @maliasadi!It's done in the `cpptestswithOpenBLASNoLAPACK` job.